src/App.tsx

```diff
@@ -1,24 +1,203 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import { Todo } from './types/Todo';
+import { Error } from './types/Error';
+import { FilterTodos } from './types/FilterTodos';
+import {
+  getTodos,
+  createTodo,
+  deleteTodo,
+  updateTodo,
+} from './api/todos';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
 
-const USER_ID = 0;
+const USER_ID = 6550;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState(Error.NONE);
+  const [filterBy, setFilterBy] = useState(FilterTodos.ALL);
+  const [activeInput, setActiveInput] = useState(true);
+
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [idUpdating, setIdUpdating] = useState([0]);
```

Comment:
why do we need to keep 0 here?
```suggestion
  const [updatingIds, setUpdatingIds] = useState([0]);
```

Reply:
this is from previous task **immediately after sending a request create a todo with `id: 0`**

Reply:
How is this connected? You should set temp todo on adding, but it is an array with updatingIds. If you have questions or don't understand it clearly, ask in the chat

Reply:
if i leave here 0, only then i have loading animation when i add todo